x-pack/plugins/security_solution/public/timelines/components/timeline/tabs_content/index.tsx

```diff
@@ -140,7 +151,8 @@ const ActiveTimelineTab = memo<ActiveTimelineTabProps>(
     );
 
     const isGraphOrNotesTabs = useMemo(
```

Comment:
We should probably make this more descriptive in a follow up since it's scope has expanded. Or add a comment to it's use case